feat: integrate onboarding data and demos (#473)
algolia/doc-code-samples: README.md

# doc-code-samples

> This repository hols all code examples shown on the [Algolia.com documentation website](https://algolia.com/doc).

Reviewer: ```suggestion
This repository holds all code examples shown on the [Algolia.com documentation website](https://algolia.com/doc).
```

> This repository hols all code examples shown on the [Algolia.com documentation website](https://algolia.com/doc).

Reviewer 2: all runnable? because we don't host the snippets

Author: Sorry I'm not sure I follow. Are there sandboxes that wouldn't run on CSB?

Reviewer 2: I mean we don't host the non-sandboxes here

Author: Thanks for the clarification. I've updated the readme to be more precise in regards to what the repository contain.

## InstantSearch code samples

> All code samples described in InstantSearch guides are runnable via [CodeSandbox](https://codesandbox.io/). The full list can be easily viewed at this page: https://instantsearch-templates.netlify.com.

Reviewer: ```suggestion
All code samples in InstantSearch guides run on [CodeSandbox](https://codesandbox.io/). The full list is available [here](https://instantsearch-templates.netlify.com).
```
